### ISSUE TITLE
feat(data, tags): add tags.json

### ISSUE THREAD

**Issue by safiler** [Labx/LibraryBox-app#9]
```
[
    {
        "name": "Historique",
        "id": 8
        "books": [
        {
       "sort": "Idiot - Tome I, L'",
       "title": "L'Idiot - Tome I",
       "id": 8
    },
]
}
]
```

**Comment by edouard-lopez** [comment]
fixed by ace39c8c8, see [app/data/tags.json](/Labx/LibraryBox-app/blob/master/app/data/tags.json).

**Comment by edouard-lopez** [comment]
JSON schema is described in [submodule LibraryBox-metadata/tags/squash-tags-data.jq
](https://github.com/Labx/LibraryBox-metadata/blob/master/tags/squash-tags-data.jq).
